docs/data/data-grid/column-dimensions/column-dimensions.md

```diff
@@ -58,6 +58,45 @@ To capture changes in the width of a column there are two callbacks that are cal
 - `onColumnResize`: Called while a column is being resized.
 - `onColumnWidthChange`: Called after the width of a column is changed, but not during resizing.
 
+## Autosizing [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+
+`DataGridPro` allows to autosize the columns' dimensions based on their content. Autosizing is enabled by default. To turn it off, pass the `disableAutosize` prop to the datagrid.
```

Comment:
> Autosizing is enabled by default.

This is a bit misleading since it could be understood as "columns are autosized by default", which is not the case.
Should we rephrase it to "Double-clicking a column header separator is enabled by default"?
@romgrk

Reply:
Sure, that sounds good.